themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -71,52 +71,58 @@
-            {% else %}
-              <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-nav" aria-controls="navbar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-              </button>
-              <nav class="navbar navbar-expand-lg main-navbar px-0 collapse ml-auto flex-column d-lg-flex" id="navbar-nav">
-                <div class="navbar-inner d-flex">
-                  {% if page.secondary_menu %}
-                    <div class="secondary-menu-region">{{ page.secondary_menu }}</div>
-                  {% endif %}
-                  {% if page.primary_menu %}
-                    <div class="menu-region">{{ page.primary_menu }}</div>
-                  {% endif %}
-                  {% if  page.lang_menu %}
-                    <div class="lang-region">{{ page.lang_menu }}</div>
-                  {% endif %}
-                </div>
-              </nav>
-            {% endif %}
+{% set header_classes = [
+  'container',
+  'no-padding-md',
+  is_cta_header ? 'hanging-header',
+  page.admin_tabs ? 'alert-ladning-header',
+] %}
+
+{# Header section #}
+{% set header %}
+  {% block head %}
+    <header id="header" class="header" role="banner"
+            aria-label="{{ 'Site header'|t }}">
+      <nav{{ navbar_attributes.addClass(classes) }}>
+        <div class="{{ header_classes | join(' ') }}">
+          <div class="navbar-brand d-flex align-items-center">
+            {{ page.header }}
           </div>
-        </nav>
-      {% endif %}
-    {% endblock %}
-  </header>
+          {% if is_cta_header %}
+            {% if cta_button is not empty %}
+              <div class="d-content-cta-wrapper">
+                {{ cta_button | render }}
+              </div>
+            {% endif %}
+          {% else %}
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-nav" aria-controls="navbar-nav" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <nav class="navbar navbar-expand-lg main-navbar px-0 collapse ml-auto flex-column d-lg-flex" id="navbar-nav">
+              <div class="navbar-inner d-flex">
+                {% if page.secondary_menu %}
+                  <div class="secondary-menu-region">{{ page.secondary_menu }}</div>
+                {% endif %}
+                {% if page.primary_menu %}
+                  <div class="menu-region">{{ page.primary_menu }}</div>
+                {% endif %}
+                {% if  page.lang_menu %}
+                  <div class="lang-region">{{ page.lang_menu }}</div>
+                {% endif %}
+              </div>
+            </nav>
+          {% endif %}
+        </div>
+      </nav>
+    </header>
+  {% endblock %}
+{% endset %}
+
+<div id="page-wrapper" class="page-wrapper {% if is_cta_header %}landing-page-header{% endif %}">
```

Comment:
Może by tu też `Attributes`? Wiem, że to może być trochę overkill, ale to akurat szablon, który ludzie będą często zmieniać.